data/README.md

```diff
@@ -71,12 +71,24 @@ Function used to subscribe to data changes. The listener function is called each
 // Subscribe.
 const unsubscribe = wp.data.subscribe( () => {
 	const data = {
-		slug: wp.data.select( 'core/editor', 'getEditedPostSlug' ),
+		//Post title
+		title: wp.data.select( 'core/editor', 'getEditedPostAttribute', 'title' ),
+		//Post slug
+		slug: wp.data.select( 'core/editor', 'getEditedPostAttribute', 'slug' ),
+		//Post type
+		type: wp.data.select( 'core/editor', 'getEditedPostAttribute', 'type' ),
+		//Post content
+		content :wp.data.select( 'core/editor', 'getEditedPostAttribute', 'content' ),
+		//Number of blocks currently selected
+		selectedBlockCount: wp.data.select( 'core/editor', 'getSelectedBlockCount' ),
```

Comment:
The Documentation of the data module is meant to be generic. Should we document these selectors as part of the `editor` module's documentation instead?

Also, in Calypso we have a script to generate selectors documentation automatically, we might be able to adapt it here cc @aduth